tools/MarkdownConverter/Converter/MarkdownSourceConverter.cs

```diff
@@ -424,6 +423,26 @@ IEnumerable<OpenXmlCompositeElement> Paragraph2Paragraphs(MarkdownParagraph md)
                 {
                     var mdcell = mdrow[icol];
                     var cell = new TableCell();
+
```

Comment:
I freely admit that this is ugly.

But it works.

Any suggestions welcome.  I did try a pattern match and that was less readable.